docs/code_gen_options.rst

```diff
@@ -149,21 +148,49 @@ as a function argument, for instance, the length is unknown.
 Debugging Options
 -----------------
 
-It may be desirable to have access to debug information regarding the contract execution. 
-However, this will lead to more instructions as well as higher gas usage. Hence, it is disabled by default,
-but can be enabled using CLI flags. Just make sure not to use them in production deployments.
+It is desirable to have access to debug information regarding the contract execution in the testing phase.
+Therefore, by default, debugging options are enabled; however, they can be deactivated by utilizing the command-line interface (CLI) flags.
+Debugging options should be disabled in release builds, as debug builds greatly increase contract size and gas consumption.
+Solang provides three debugging options, namely debug prints, logging API return codes, and logging runtime errors. For more flexible debugging,
+Solang supports disabling each debugging feature on its own, as well as disabling them all at once with the ``--release`` flag.
 
-.. _log-api-return-codes:
+.. _no-print:
+
+Print Function
+++++++++++++++
+
+Solang provides a `print builtin <https://solang.readthedocs.io/en/latest/language/builtins.html#print-string>` which is enabled by default.
```

Comment:
For references inside the same document, we do not use links. We use cross-references, like these ones:

https://github.com/hyperledger/solang/blob/242c187fa981b523102c0147f4ed272836c01f69/docs/language/managing_values.rst?plain=1#L46

https://github.com/hyperledger/solang/blob/242c187fa981b523102c0147f4ed272836c01f69/docs/language/types.rst?plain=1#L163

Reply:
This is important to maintain coherence if we someday change the website domain or the organization of the docs. A cross-reference will never brake.